platform/ios/src/MGLMapView.mm

```diff
@@ -3536,17 +3532,7 @@ - (void)setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count:
                 count,
                 NSStringFromUIEdgeInsets(insets),
                 MGLStringFromBOOL(animated));
-    [self setVisibleCoordinates:coordinates count:count edgePadding:insets direction:self.direction animated:animated];
-}
-
-- (void)setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count:(NSUInteger)count edgePadding:(UIEdgeInsets)insets direction:(CLLocationDirection)direction animated:(BOOL)animated
-{
-    MGLLogDebug(@"Setting: %lu coordinates edgePadding: %@ direction: %f animated: %@",
-                count,
-                NSStringFromUIEdgeInsets(insets),
-                direction,
-                MGLStringFromBOOL(animated));
-    [self setVisibleCoordinates:coordinates count:count edgePadding:insets direction:direction duration:animated ? MGLAnimationDuration : 0 animationTimingFunction:nil];
+    [self setVisibleCoordinates:coordinates count:count edgePadding:insets direction:self.direction duration:animated ? MGLAnimationDuration : 0 animationTimingFunction:nil];
```

Comment:
nit: self.direction is the current value, not the new?

Reply:
Yes, in this version of the method, no new `direction` is passed in, so we keep the current one. Ideally this would say −1, but #14574 incorrectly clamps negative values to 0.

Reply:
Ah, I overlooked the signature change in the changeset.